lib/Graph/Nodes.cpp

```diff
@@ -296,8 +296,13 @@ static bool verifyRelu(NodeValue result, NodeValue input) {
 
 static bool verifyRegression(NodeValue src, NodeValue dest,
                              NodeValue expected) {
-  return checkSameType(src, dest, dest.getNode()) &&
-         checkSameType(dest, expected, dest.getNode());
+  const Node *parent = dest.getNode();
```

Comment:
this seems unrelated to the PR.

Reply:
It was actually required for MLTest's `testFindPixelRegression` test to pass. It quantizes a regression node, which is eventually lowered into nothing during inference but is there for training. I could move it to another PR if desired.